From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@neilpeterson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (nepeters). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

wmgries committed:
Update supported version table to include missed versions.

gled4er committed:
Adding clarification for completion condition of StartNew Async.

gled4er committed:
Extending the definition of GetStatusAsync with information about retrieveing the execution history.

gled4er committed:
Adding additional details for GetStatusAsync when calleda as HTTP API.

gled4er committed:
Adding documentation for WaitForCompletionOrCreateCheckStatusResponseAsync method.

gled4er committed:
Adding documentation page for unit testing Durable Functions.

fauhse committed:
Correcting the update policy statements

fauhse committed:
addressing auto-feedback

fauhse committed:
attempting to fix link + layout issues

fauhse committed:
fixing ordered list

fauhse committed:
fixing another ordered list style issue.

gled4er committed:
Improving style in durable-functions-http-api page and removing section for sync response.

gled4er committed:
Fixing typo and moving wait for orchestrator completion section to instance management article.

gled4er committed:
Modifying the strucutre and contents of durable-functions-unit-testing article.

gled4er committed:
Improving style and fixing minor issues.

gled4er committed:
Minor style check.

fauhse committed:
addressing Will's feedback and a point from Adam

itechedit committed:
edit pass: custom-decision-service-experimentation-reference

gled4er committed:
Improving unit testing document.

kadimitr committed:
Merge branch 'master' into kaniodimitrov/durable-functions-beta-3

lenadroid committed:
Spark on AKS doc

lenadroid committed:
correct abbreviation

kadimitr committed:
Addressing feedback from code review iteration.

lenadroid committed:
Styling changes

lenadroid committed:
jar name consistency and date

lenadroid committed:
Merge branch 'master' of github.com:MicrosoftDocs/azure-docs-pr

lenadroid committed:
Merge branch 'master' of github.com:MicrosoftDocs/azure-docs-pr

lenadroid committed:
Merge branch 'master' of github.com:MicrosoftDocs/azure-docs-pr

hpsin committed:
Feedback

lenadroid committed:
File name update, links and ACR note

lenadroid committed:
Merge branch 'master' of github.com:MicrosoftDocs/azure-docs-pr

kadimitr committed:
Removing sample code and referencing code in the samples project.

kadimitr committed:
Addressing issues find by Chris in last review iteration.

mmacy committed:
[ACI] expand log + add attach

mmacy committed:
[ACI] add logs and attach to T-shooting

mmacy committed:
[ACI] extra space

kadimitr committed:
Fixing issues from the last review iteration. Improving style, adding commas, adding links for follow up steps and removeing redundant text.

mmacy committed:
[ACI] grammar mods

mmacy committed:
[ACI] simplification

mmacy committed:
[ACI] more simplification

lenadroid committed:
Updates for prereqs, note on VM sizes, and ACR auth comment

wmgries committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into SyncUpdatePolicy

lenadroid committed:
Merge branch 'master' of github.com:MicrosoftDocs/azure-docs-pr

wmgries committed:
Clean up of Agent Update policy.

wmgries committed:
Minor tweaks to formatting.

wmgries committed:
Minor fixes.

gled4er committed:
Adding line breaks between links for next actions.

gled4er committed:
Removing references to fluent assertions in the documentation.

wmgries committed:
Fix dates.

mmacy committed:
[ACI] windows caching + inline TOC

hpsin committed:
Casing on the bookmark link

ecfan committed:
Add new quickstart content

ecfan committed:
Add + update art links

ecfan committed:
Refactored ToC + manage logic apps with VS content

ecfan committed:
Add anchor link to portal quickstart

Commit:
Update article based on UI changes

ecfan committed:
Refactor manage with VS topic + update links

ecfan committed:
Update ToC for renamed topic

ecfan committed:
Update redirection file

itechedit committed:
edit pass: active-directory-saas-box-tutorial

ecfan committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into mvc-logic-apps-vs

wmgries committed:
Revert supported until for AFS.

itechedit committed:
Update active-directory-saas-box-tutorial.md

bwren committed:
toc

gled4er committed:
Adding Azure name to meta title and lower casing unit testing.

gled4er committed:
Adding entry to TOC file for the unit testing article.

ecfan committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into mvc-logic-apps-vs

msjuergent committed:
ops_doc1

kadimitr committed:
Moving the location of the unit testing article.

msjuergent committed:
ops_doc2

msjuergent committed:
ops_doc3

Commit:
Update screenshot

v-geberr committed:
Split interactive testing from batch testing on concept and how-to

v-geberr committed:
test concept - remove batch testing

mmacy committed:
[ACI] remove not-running bit

Commit:
first batch of stalness edits for march 2018

mmacy committed:
[ACI] typo fix

v-geberr committed:
batch testing how-to

v-geberr committed:
moving images around

v-geberr committed:
toc changes

ecfan committed:
Address PM feedback

v-geberr committed:
wip

ecfan committed:
Tweaks

ecfan committed:
Fix toc files

v-geberr committed:
wip

ecfan committed:
Fix ToC

ecfan committed:
Refactoring redirected content

ecfan committed:
Fix ToC error

ecfan committed:
Revert redirected file

sharonlo101 committed:
added timeout information

v-geberr committed:
concept batch test

v-geberr committed:
put endpoint testing back in

v-geberr committed:
put quota info back

mumian committed:
format

ecfan committed:
Rename author

ecfan committed:
Restore deleted files

ecfan committed:
Update redirects and manager

ecfan committed:
Remove redirected files

v-geberr committed:
link fixes

v-geberr committed:
train how-to

v-geberr committed:
remove train from how-to test

v-geberr committed:
add missing link

v-geberr committed:
edits

neilpeterson committed:
nepeters updates

lenadroid committed:
Merge pull request #1 from neilpeterson/patch-2

nepeters updates

j-martens committed:
AML: Studio deprecating live chat

v-geberr committed:
edits

neilpeterson committed:
Updates from @neilpeterson

* images

v-geberr committed:
edits

j-martens committed:
AML: Studio removes live chat feature

Commit:
automation: update run as documentation

v-geberr committed:
edits

v-geberr committed:
edits

Commit:
updates

ecfan committed:
Address PM feedback

v-geberr committed:
edits

v-geberr committed:
edits

v-geberr committed:
edits

neilpeterson committed:
ACR auth steps (#3)

Commit:
corrected  bookmark

bwren committed:
Removeals, renames, and redirects for legacy log search

KumudD committed:
added draft

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into outboundconnectionsclassic

bwren committed:
redirection file

GitHubber17 committed:
Update spark-job.md

Commit:
hybrid01

bwren committed:
Merge branch 'master' into oms-ga

v-geberr committed:
change title

KumudD committed:
updated TOC

GitHubber17 committed:
Merge pull request #34875 from lenadroid/master

Adding a doc for Spark with AKS

Brenduns committed:
Table updates

Commit:
Merge pull request #35026 from mmacy/aci-attach

[ACI] expand logs section + add attach

Lisaco88 committed:
Fixed typo

JasonWHowell committed:
Update tutorial-design-database-using-cli.md

JasonWHowell committed:
Update tutorial-design-database-using-portal.md

GitHubber17 committed:
Merge pull request #35352 from sharonlo101/patch-1

added timeout information

Lisaco88 committed:
Merge pull request #35314 from v-geberr/0314-testing-howto-separation

Split interactive testing from batch testing on concept and how-to

bwren committed:
Update toc.yml

mmacy committed:
[ACI] link fix

PRMerger14 committed:
Merge pull request #35380 from JasonWHowell/patch-2

MySQL: Update tutorial-design-database-using-portal.md

ecfan committed:
Refactoring tweaks

JasonWHowell committed:
Update tutorial-design-database-using-azure-cli.md

JasonWHowell committed:
Update tutorial-design-database-using-azure-portal.md

JasonWHowell committed:
Update tutorial-design-database-using-azure-cli.md

Commit:
quickfix03

johnkemnetz committed:
add partners

johnkemnetz committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into add-partner

mimckitt committed:
Fixing C# code samples

Fixed 3 instances of code samples where "public" was not included before the class. This was causing issues when customers were running this sample in a copy-paste mode.

PRMerger19 committed:
Merge pull request #35381 from mmacy/aci-troubleshoot-link-fix

[ACI] troubleshooting link fix

rjagiewich committed:
Merge pull request #35287 from bwren/oms-ga

Log Analytics TOC - Remove legacy log search

PRMerger20 committed:
Merge pull request #35382 from JasonWHowell/patch-2

PostgreSQL: Update tutorial-design-database-using-azure-cli.md

Commit:
Merge pull request #35384 from MarkusVi/quickfix03

quickfix03

j-martens committed:
putting filename back

PRMerger12 committed:
Merge pull request #35387 from MicahMcKittrick-MSFT/patch-7

(ForumsTeam) Fixes MicrosoftDocs/azure-docs#5769

PRMerger12 committed:
Merge pull request #34922 from hpsin/RevokeToken2

Add Token Revocation reasons

v-geberr committed:
Non-blocking fix requested

Commit:
hybrid01

ecfan committed:
Update for redirected files

Commit:
hybrid01

bandersmsft committed:
azure policy - cli quickstart minor fix

ecfan committed:
Clean up art

bandersmsft committed:
azure policy - overview minor fix

PRMerger20 committed:
Merge pull request #35375 from MarkusVi/hybrid01

hybrid01

ecfan committed:
Fix links and other tweaks

PRMerger19 committed:
Merge pull request #35379 from JasonWHowell/patch-1

MySQL: Update tutorial-design-database-using-cli.md

ktoliver committed:
Merge pull request #34178 from gled4er/kaniodimitrov/durable-functions-beta-3

Adding documentation for Durable Functions Beta 3

Commit:
Merge pull request #35383 from JasonWHowell/patch-3

PostgreSQL: Update tutorial-design-database-using-azure-portal.md

Commit:
Merge pull request #35392 from bandersmsft/master

azure policy - minor updates

ktoliver committed:
Merge pull request #35385 from johnkemnetz/add-partner

Add partner

Commit:
Merge pull request #35304 from msjuergent/hanaopsguide

ops_doc1

j-martens committed:
AML: Tutorial update for usability testing

ecfan committed:
Content cleanup

KumudD committed:
fixed edits

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into outboundconnectionsclassic

Commit:
first batch of stalness edits for march 2018 -2

Commit:
fixing code sample

Commit:
Acrolynx edits

Commit:
Acrolynx edits 2

rwike77 committed:
added note about Windows 2016 for containers

Commit:
Merge pull request #35394 from bradsev/staleness-edits-2018-3-14

Staleness edits 2018-3-14

Commit:
Merge pull request #35397 from rwike77/containerupdate

added note about Windows 2016 for containers

christiankuhtz committed:
Tweaks

KumudD committed:
Merge pull request #2 from christiankuhtz/patch-79

Tweaks

dominicbetts committed:
Clarify details of feedback message

PRMerger18 committed:
Merge pull request #35405 from dominicbetts/clarify-c2d

Clarify details of feedback message

Commit:
Merge pull request #35357 from mumian/0314-powerbi

format

PRMerger16 committed:
Merge pull request #35393 from v-geberr/0314-batch-test-concept-cap

Non-blocking fix requested

v-geberr committed:
more locations of timezoneOffset explanation

Brenduns committed:
table update

Commit:
Merge pull request #35407 from Brenduns/vmupdates

Vmupdates

PRMerger14 committed:
Merge pull request #35409 from v-geberr/0315-timezoneoffset-fix

more locations of timezoneOffset explanation

Brenduns committed:
More corrections from the PM

Commit:
Merge pull request #35411 from Brenduns/vmupdates

More corrections from the PM

v-geberr committed:
fix for utterance limit - deployed today on LUIS

itechedit committed:
Merge pull request #35256 from ecfan/mvc-logic-apps-vs

Create quickstart from existing VS content about creating logic apps and refactor managing with VS

CarlRabeler committed:
Update sql-database-servers-databases.md

Commit:
corrected formatting

v-geberr committed:
edits

itechedit committed:
Merge pull request #35376 from KumudD/outboundconnectionsclassic

Outbound connections classic

Commit:
Merge pull request #35415 from CarlRabeler/patch-26

Update sql-database-servers-databases.md

ShawnJackson committed:
Merge pull request #34752 from itechedit/custom-decision-service-experimentation-reference

edit pass: custom-decision-service-experimentation-reference

Commit:
Merge pull request #35414 from v-geberr/0315-boundaries

fix for utterance limit - deployed today on LUIS

Commit:
Merge pull request #35122 from wmgries/SyncUpdatePolicy

Fix sync update policy

itechedit committed:
Merge pull request #35266 from craigcaseyMSFT/v-craic0313

Update article based on UI changes

CarlRabeler committed:
Update az-overview.md

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

itechedit committed:
Merge pull request #35371 from georgewallace/autorunasupdates

automation: update run as documentation

PRMerger15 committed:
Merge pull request #35418 from CarlRabeler/patch-27

Update az-overview.md

ShawnJackson committed:
Merge pull request #35270 from itechedit/active-directory-saas-box-tutorial

edit pass: active-directory-saas-box-tutorial

ktoliver committed:
Merge pull request #35367 from j-martens/master

ML: Studio removes live chat feature

v-alje committed:
Merge pull request #35420 from MicrosoftDocs/FromPublicRepo

From public repo
